### ISSUE TITLE
Object doesn't support property of method 'forEach' -  IE 11

### ISSUE THREAD

**Issue by blandfried-magellan** [AndrewRedican/react-json-editor-ajrm#26]
1. What version of RJEA are you using (react-json-editor-ajrm version)?
   - 2.4.9
2. What operating system and processor architecture are you using?
  -macOS High Sierra
3. What did you do?
  -Tried to edit JSON in Internet Explorer 11
4. What did you expect to see?
 - The onChange function to fire correctly with changed JSON
5. What did you see instead?
 - Error in console about forEach not being supported

I've seen this before where the project isn't getting transpiled correctly with polyfills.

![screen shot 2018-07-24 at 11 48 26 am](https://user-images.githubusercontent.com/38441047/43150577-3ece43ee-8f38-11e8-81b0-f6bb32334883.png)

**Comment by AndrewRedican** [comment]
@blandfried-magellan, good catch. I'll submit a fix in the next few days.

**Comment by blandfried-magellan** [comment]
I think you want to use transformRuntime plugin. Had same issue with below and that's how they fixed it.

https://github.com/okta/okta-signin-widget/commit/3b58e80cef75bffe2acfc253e7e9221c6f6b31fa

**Comment by PatrickSachs** [comment]
I did some more readup on this today since this issue still persists:

We are already using @babel/transform-runtime which was suggested above in our current build setup. This plugin polyfills ES6 features to ES5.

The reason that it still isn't working, is that the transform-runtime plugin does **not** polyfill instance methods (such as forEach, map, reduce, ...), due to the dynamic nature of JavaScript(It cannot possibly know if myVar.forEach refers to the built-in array method, or if we simply have a custom object with a forEach property on it). 
See this comment and/or post for details: https://github.com/w3ctag/polyfills/issues/6#issuecomment-272177846

So what is the solution to this? Honestly, I'm not certain. 

We could go the route the mentioned okta-signin-widget took and add another polyfill dependency to this project. The babel-polyfill library is _to my understanding_ though rather large and injects itself into the global scope since it includes every polyfill possibly imaginable to ensure that all instance methods exists.

We could write our own functions to emulate the functionality of forEach, etc... But I'm not sure of how maintainable it is to manually check browser support every time you use a function, and possibly re-write your own version of it.

We could no longer polyfill at all and require the user to do it, but this will make this library rather unintuitive to use and probably create a large amount of issues.

Any suggestions? 😃

**Comment by PatrickSachs** [comment]
> How do you feel about just sticking to plain old for loops?

Yeah, I think that's the easiest, least headache inducing option for now.
